features2d.go

```diff
@@ -795,3 +795,85 @@ func (d *SIFT) DetectAndCompute(src Mat, mask Mat) ([]KeyPoint, Mat) {
 
 	return getKeyPoints(ret), desc
 }
+
+// DrawMatches draws matches on combined train and querry images.
+//
+// For further details, please see:
+// https://docs.opencv.org/3.4/d4/d5d/group__features2d__draw.html
```

Comment:
This should be pointing to https://docs.opencv.org/master/d4/d5d/group__features2d__draw.html#gad8f463ccaf0dc6f61083abd8717c261a